.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,38 @@
+# PR Template
+
+Thank you for your contribution! To help us review your pull request effectively, please make sure to include the following details in your PR description.
+
+## 📄 What does this PR do?
+
+Describe your changes clearly and concisely.
+
+Example:
+- Refactors video card layout for better accessibility
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)